float64[6] cartesian_stiffness
---
bool success
string error

